docs/references/studio/api/datasets.md

```diff
@@ -0,0 +1,257 @@
+
+### List Datasets
+
+**Description**: Retrieve a list of datasets with optional prefix filtering.
+
+**Endpoint**: `/datachain/datasets`
+
+**Method**: `GET`
+
+**Authentication**: Requires token with `read` access to `DATASETS` scope from [`datachain auth login`](../../../commands/auth/login.md)
+
+**Parameters**:
+
+- `prefix` (query, optional): Filter datasets by name prefix
+- `team_name` (query, required): Team identifier (automatically added by client)
+
+**Example Request**:
+```bash
+curl -X GET "https://studio.datachain.ai/api/datachain/datasets?prefix=training&team_name=my-team" \
+  -H "Authorization: token YOUR_STUDIO_TOKEN" \
+  -H "Content-Type: application/json"
+```
+
+**Example Response**:
+```json
+[
+  {
+    "attrs": ["training"],
+    "created_at": "2025-06-11T14:22:31.123456+00:00",
+    "created_by_id": 2,
```

Comment:
This is not the case for documentation, but in terms of API should we improve this field?
E.g.:
```json
    "created_by": {
        "id": 2,
        "name": "User name"
    }
```
or something like that 🤔